sources/models/src/modeled_types/shared.rs

```diff
@@ -26,10 +26,10 @@ pub struct ValidBase64 {
 }
```

Comment:
This has the same question as in `kubernetes.rs` should we leave `TryFrom` and add `FromStr` instead of completely replacing it?

Reply:
I'll leave both `TryFrom` and `FromStr` for the following mentioned `pub struct`s:

* `sources/models/src/modeled_types/kubernetes.rs`:
    * `KubernetesName`
    * `KubernetesLabelKey`
* `sources/models/src/modeled_types/shared.rs`
    * `ValidBase64`

Is that correct?

Reply:
If possible, it would be good to have some comments on why/when to use one or the other. That way when someone (me) works on this months from now there isn't any confusion about the correct one to use.

Reply:
After further consideration, I agree with @arnaldo2792 in that only `FromStr` should be available:
https://github.com/bottlerocket-os/bottlerocket/pull/2298#discussion_r967393727